frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1032,9 +1033,11 @@
                         else if (details.remaining_submissions_today_count > 0) {
                             vm.remainingSubmissions = details;
                             vm.showSubmissionNumbers = true;
+                            vm.submissionsExhausted = false;
```

Comment:
Once you initialize it with `false`, you don't need this line. Please remove it.

Reply:
Initialised with `True`, so required and to switch from 
- When Submissions are remaining **and** `Upload File` is selected  

**to** 
- When Submission Limit is exhausted